core/storage/suggestion/gae_models.py

```diff
@@ -485,6 +485,28 @@ def get_in_review_translation_suggestions(
             cls.language_code.IN(language_codes)
         )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
 
+    @classmethod
+    def get_in_review_translation_suggestions_by_exp_ids(
+        cls, exp_ids: List[str], language_code: str
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all in review translation suggestions matching the supplied
+        exp_ids and language_code.
+
+        Args:
+            exp_ids: list(str). Exploration IDs.
+            language_code: str. Language code.
+
+        Returns:
+            list(SuggestionModel). A list of suggestions matching the supplied
+            exp_ids and language_code.
+        """
+        return cls.get_all().filter(datastore_services.all_of(
```

Comment:
Does an entry already exist in index.yaml for this? If not, we should add it, otherwise we'll run into problems on production.

Reply:
I think, since the Python 3 update, we've lost the autogen functionality :( Normally they should be autogenerated when the local dev server tries to make that query.

@vojtechjelinek -- could you please weigh in on this? I am worried that things like this won't get caught until release time, at which point we'll have a scramble -- we should put some measures in place to avoid it (at the very least, tell contributors to update index.yaml manually in cases where it's needed, if we can't autogenerate).

Reply:
We have a script for this now, so if the query is executed while running local devserver and the the devserver is properly stopped (using Ctrl+C) the new entry should be added to index.yaml.

Reply:
Hm -- but this didn't happen for @sagangwee (is that right, Sagang)?

Reply:
Can you please provide more context? Can you try to run the devserver, execute this code, look what is the contents of _../cloud_datastore_emulator_cache/WEB-INF/index.yaml_ and then stop the devserver it using Ctrl+C. After that send me the whole output of the devserver in the terminal.

Reply:
@vojtechjelinek 
The index contained:

```
- kind: "QuestionSkillLinkModel"
  properties:
  - name: "skill_id"
  - name: "last_updated"
    direction: desc
- kind: "GeneralSuggestionModel"
  properties:
  - name: "author_id"
  - name: "deleted"
  - name: "suggestion_type"
  - name: "created_on"
    direction: desc
- kind: "ExplorationOpportunitySummaryModel"
  properties:
  - name: "incomplete_translation_language_codes"
  - name: "topic_name"
```

and the terminal output can be found here: https://gist.github.com/sagangwee/34f65264b4ef5e1b49eee4ed4fb03b7d

Reply:
@sagangwee these indexes were already there in `index.yaml` so there was no need to extend it.
for your reference (some screenshots from the index.yaml(not from webinf)):

![image](https://user-images.githubusercontent.com/69108486/152310253-8426fb1f-7b66-4855-83f2-3be01f5b4e9b.png)
![image](https://user-images.githubusercontent.com/69108486/152310292-cd3645ee-95c5-4aa8-9a6b-1f5e93364aa3.png)
![image](https://user-images.githubusercontent.com/69108486/152310354-df90a7e3-494d-4a35-acd0-006c54acbba4.png)
or did i go wrong somewhere ??

Reply:
also, I think with `"` and without it will perform the same.

Reply:
Hmm, interesting, the thing is that this query contains other fields that are not in the _index.yaml_ like `status`, `suggestion_type` and `language_code`. It seems like the datastore didn't generate the indexes correctly. I think we should test this PR on the backup server.

@sagangwee When you were running this query were there any suggestions to fetch (just checking what might be wrong)?